src/main/java/org/ozwillo/dcexporter/controller/DcModelMappingController.java

```diff
@@ -5,26 +5,29 @@
 import org.ozwillo.dcexporter.dao.DcModelMappingRepository;
 import org.ozwillo.dcexporter.model.DcModelMapping;
 import org.ozwillo.dcexporter.service.CkanService;
+import org.ozwillo.dcexporter.service.DcModelMappingService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 import static org.springframework.web.bind.annotation.RequestMethod.POST;
 
 @RestController
-@RequestMapping("/api/dc-model-mapping")
+@RequestMapping("/api/dc-exporter")
```

Comment:
When I added the logs, I respected the naming of other mapping "api / ckan", "api / dc". It's not good ?